src/bun.js/api/bun.zig

```diff
@@ -486,6 +486,18 @@ pub fn getImportedStyles(
     return JSValue.createStringArray(ctx.ptr(), styles.ptr, styles.len, true).asRef();
 }
 
+pub fn newOs(
+    _: void,
+    ctx: js.JSContextRef,
+    _: js.JSObjectRef,
+    _: js.JSObjectRef,
+    args: []const js.JSValueRef,
+    _: js.ExceptionRef,
+) js.JSValueRef {
+    const is_windows = args.len == 1 and JSValue.fromRef(args[0]).toBoolean();
+    return Node.Os.create(ctx.ptr(), is_windows).asObjectRef();
```

Comment:
I think OS won't need to allocate any memory besides creating the JS object. Does it need to take a `is_windows` param?